core/ratarmountcore/factory.py

```diff
@@ -41,6 +42,32 @@ def openMountSource(fileOrPath: Union[str, IO[bytes]], **options) -> MountSource
                 [(lambda file=file: open(file, 'rb')) for file in filesToJoin]  # type: ignore
             )
 
+    forceLibarchive: bool = options.get("forceLibarchive", False)
+    special_formats = ('zip', 'tar', 'rar') if not forceLibarchive else (None, )
```

Comment:
What happens with xz, gzip, bz2, which are also seek-accelerated by SQLiteIndexedTar, and their tar variants (.tar.xz, ...)?